sdk/core/azure-core/test/e2e/azure_hang.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+
+#include <azure/storage/blobs/blob.hpp>
+
+int main()
+{
+  using namespace Azure::Storage::Blobs;
+
+  std::string containerName = "sample-container";
+  std::string blobName = "sample-blob";
+  std::string blobContent;
+  blobContent.resize(50 * 1024ULL * 1024, 'x');
+
+  auto containerClient = BlobContainerClient::CreateFromConnectionString(
+      std::getenv("STORAGE_CONNECTION_STRING"), containerName);
+  try
+  {
+    containerClient.Create();
+  }
+  catch (std::runtime_error& e)
+  {
+    // The container may already exist
+    std::cout << e.what() << std::endl;
```

Comment:
Should we just stop and return here, on error?

Reply:
not for this test, we want to make the upload even if container already exists.

Reply:
We should error out in cases that Create was something other than blob already exists.  Its wrong to continue if we get a permission issue, network failure, etc...